tests/test-cmd-copy-texture.cpp

```diff
@@ -13,7 +13,13 @@ using namespace rhi::testing;
 GPU_TEST_CASE("cmd-copy-texture-full", D3D12 | Vulkan | WGPU)
 {
     TextureTestOptions options(device);
-    options.addVariants(TTShape::All, TTArray::Both, TTMip::Both, TTMS::Both);
+    options.addVariants(
+        TTShape::All,    // all shapes
+        TTArray::Both,   // array and none-array
```

Comment:
`non-array`

_none_ means "nothing" or "zero" which I don't think is correct here .. I think I've seen this non/none swap in other places before, maybe worth a quick text search.